Tools/CISupport/ews-app/ews/common/buildbot.py

```diff
@@ -20,8 +20,6 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from future.utils import lrange
```

Comment:
This change is similar to https://github.com/buildbot/buildbot/commit/12a6169edddfd139d094037a3299b4156a5f711f#diff-c01b56d05eef0a71975a3da7d5c7a190bba2cd0b2b91cf81f17f61cadb6291cc